pragma-sdk/pragma_sdk/onchain/client.py

```diff
@@ -150,23 +139,12 @@ def _setup_contracts(self):
             provider=provider,
             cairo_version=1,
         )
-
-    async def get_balance(self, account_contract_address, token_address=None) -> int:
-        """
-        Get the balance of an account given the account contract address and token address.
-
-        :param account_contract_address: The account contract address.
-        :param token_address: The token address. If None, will use ETH as the token address.
-        :return: The balance of the account.
-        """
-
-        client = Account(
-            address=account_contract_address,
-            client=self.client,
-            key_pair=KeyPair.from_private_key(1),
-            chain=CHAIN_IDS[self.network],
+        self.summary_stats = Contract(
```

Comment:
not sure we want to do it with the others since most of the time it's not needed

Reply:
ah yeah ; moved the function `init_summary_stats_contract` into the `SummaryStatsMixin` then